docs(linter): correct comment on what `EnablePlugins` does

overlookmotel opened the pull request:
`EnablePlugins` also allows *disabling* plugins. Correct the doc comment to reflect that.

I believe this will correct section header in docs on website (https://oxc.rs/docs/guide/usage/linter/cli.html#enable-plugins), because that file appears to be auto-generated from this comment.

Note: There are no links on website that reference `#enable-plugins` anchor, so it shouldn't break any links when this anchor changes to `#enable-disable-plugins`.

camc314 commented:
### Merge activity

* **Nov 13, 11:53 AM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.com/merges?org=oxc-project&repo=oxc) once it meets the requirements.
* **Nov 13, 11:53 AM UTC**: `camc314` added this pull request to the [Graphite merge queue](https://app.graphite.com/merges?org=oxc-project&repo=oxc).

overlookmotel committed:
docs(linter): correct comment on what `EnablePlugins` does (#15668)

`EnablePlugins` also allows *disabling* plugins. Correct the doc comment to reflect that.

I believe this will correct section header in docs on website (https://oxc.rs/docs/guide/usage/linter/cli.html#enable-plugins), because that file appears to be auto-generated from this comment.

Note: There are no links on website that reference `#enable-plugins` anchor, so it shouldn't break any links when this anchor changes to `#enable-disable-plugins`.
